be/src/common/config.h

```diff
@@ -1166,4 +1166,13 @@ CONF_mInt64(lake_local_pk_index_unused_threshold_seconds, "86400"); // 1 day
 
 CONF_mBool(lake_enable_vertical_compaction_fill_data_cache, "false");
 
+// json flat flag
+CONF_Bool(enable_json_flat, "false");
```

Comment:
which means if user want to enable this feature, they need to restart all BE node.

Reply:
Yeah, I know, I think the best way is controll by FE or load task, and should controll compaction task, but it's hard implement now